specification/paths/Collections-collection_id.json

```diff
@@ -104,8 +104,17 @@
                         }
                       },
                       "relationships": {
-                        "readOnly": true,
                         "properties": {
+                          "status": {
+                            "allOf": [
+                              {
+                                "$ref": "#/components/schemas/StatusRelationship"
+                              },
+                              {
+                                "description": "An untracked collection can be manually updated to indicate the collection has been collected."
+                              }
+                            ]
```

Comment:
This relationship is already a StatusRelationship, so the `allOf` can be replaced with just:
```
"description": "An untracked collection can be manually updated to indicate the collection has been collected."
```